dashboard/app/views/layouts/_header.html.haml

```diff
@@ -53,21 +53,21 @@
               - unless defined? @level
                 .headerlinks.hide_on_tablet{style: "margin-left: 100px"}
                   - if user_type == "teacher"
-                    %a.headerlink{:href=>"/student"}Home
-                    %a.headerlink{:href=>"/educate"}Courses
-                    %a.headerlink{:href=>"/promote"}Projects Gallery
-                    %a.headerlink{:href=>"/help"}Sections
-                    %a.headerlink{:href=>"/about"}Professional Learning
+                    %a.headerlink{:href=>"/"} Home
+                    %a.headerlink{:href=>"/courses"} Courses
+                    %a.headerlink{:href=>"/projects#gallery"} Projects Gallery
+                    %a.headerlink{:href=>CDO.code_org_url("/teacher-dashboard#/sections")} Sections
+                    %a.headerlink{:href=>"/my-professional-learning"} Professional Learning
```

Comment:
Since we have shared haml now is it not possible to have a shared file where all links are CDO.code_org_url and CDO.studio_url?